src/controllers/hooks.js

```diff
@@ -122,12 +214,13 @@ function verifyURLCandidate(baseURL) {
   }
 }
 
-function buildSlackMessage(baseURL, source, channel) {
+function buildSlackMessage(baseURL, source, hlxConfig, channel) {
+  const config = JSON.stringify(hlxConfig);
   return Message()
     .channel(channel)
     .blocks(
       Blocks.Section()
-        .text(`I discovered a new site on Edge Delivery Services: *<${baseURL}|${baseURL}>*. Would you like me to include it in the Star Catalogue? (_source:_ *${source}*)`),
+        .text(`I discovered a new site on Edge Delivery Services: *<${baseURL}|${baseURL}>*. Would you like me to include it in the Star Catalogue? (_source:_ *${source}*, _config_: *${config}*)`),
```

Comment:
please do not just print the entire object. Make it human readable:
```
I discovered a new site on Edge Delivery Services: *https://domain.com*. Would you like me to include it in the Star Catalogue? (_source:_ *CDN*, _hlx version_: *5*, _development URL_: *blah.aem.live*)
```

also, please skip hlx config related information if the source is not CDN